### PR TITLE
feat: Update Java, Ruby and Python recording instructions

### PR DESCRIPTION
**Pull request by dustinbyrne**
This brings them in line with the (already released and live in VS Code) JavaScript instructions.

Python is also updated to include changes introduced in v2 of the agent, such as running `appmap-python`. This will need some fact checking.

### CODE REVIEW

**Comment by apotterri** [comment]
Would it be possible to attach some snapshots of the updated instructions for Java and Python? I'll be happy to review them, but it's really hard to sift through all the markup surrounding them in the commits.

**Comment by dustinbyrne** [comment]
Are you able to access this page?
https://5fe18baf66f2280021e634d0-jvvulqxbcs.chromatic.com/?path=/story/pages-vs-code-install-guide-pages-record-appmaps--python-only

**Comment by apotterri** [comment]
Yup, I'll review from there.

File: packages/components/src/components/install-guide/record-instructions/Python.vue [hunk 1]
```diff
@@ -1,121 +1,231 @@
...
   },
+
+  mounted() {
+    if (!this.$refs.startCommand) return;
+    if (!(this.$refs.startCommand instanceof HTMLElement)) return;
+
+    this.typewrite(this.$refs.startCommand, [
+      '<start command>',
+      'script.py',
+      'flask --app main.app',
```
Comment (apotterri) [inline packages/components/src/components/install-guide/record-instructions/Python.vue:215]: This should be `flask --debug --app main.app`.

Comment (apotterri) [reply packages/components/src/components/install-guide/record-instructions/Python.vue:215]: Also, in the chromatic page at least, these scroll by really quickly.

**Review by apotterri — CHANGES_REQUESTED** [review]
Needs a tweak for flask, but looks good otherwise.

**Pull request merged**